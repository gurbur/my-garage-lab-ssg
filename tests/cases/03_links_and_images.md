## 링크 종합 테스트

이 문단에는 여러 종류의 링크가 있습니다.

표준 마크다운 링크는 이렇게 생겼습니다: [Google](https://google.com).
Obsidian 내부 글 링크는 이렇게 생겼습니다: [[(1) 첫 글]].
마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:

![[image_preview.png]]

링크들 사이에도 일반 텍스트가 올 수 있습니다.
