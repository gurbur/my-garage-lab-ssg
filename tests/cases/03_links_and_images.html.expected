<h2>링크 종합 테스트</h2>
<p>이 문단에는 여러 종류의 링크가 있습니다.</p>
<p>표준 마크다운 링크는 이렇게 생겼습니다: <a href="https://google.com">Google</a>. Obsidian 내부 글 링크는 이렇게 생겼습니다: <a href="(1) 첫 글">(1) 첫 글</a>. 마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:</p>
<p><img src="image_preview.png" alt="image_preview.png"></p>
<p>링크들 사이에도 일반 텍스트가 올 수 있습니다. </p>
